weaviate/collections/data.py

```diff
@@ -374,43 +414,53 @@ def replace(
 
     def update(
```

Comment:
also change of order. What is the reason?

Reply:
Same as the `replace` method

Reply:
Oh wait, I remember now why I did this one. Sebastian requested that `properties` be `Optional` when calling `update` in the case you only want to update the `vector`. So, in order to do that, `properties` must come after `uuid` since `uuid` is required. Then, given this, I changed `replace` to match `update`'s signature

Reply:
I see, that makes sense